### DISCUSSION TITLE
I modified the source code and used the local large model. Why does it say No generations found in stream?

### DISCUSSION THREAD

**Discussion by lwdnxu** [RamiAwar/dataline#365] [General]
I modified the source code and used the local large model. Why does it say  "  No generations found in stream " ? Please

**Comment by RamiAwar** [comment]
Not sure without knowing all the details. It might be that the model isn't actually returning anything for some reason, or that the model returns unexpected output structures, or other things I've never seen before.

What model are you using?

Also, we can't recommend any local models currently - the quality is pretty bad based on our testing - gpt3.5 is still our recommended (not even 4o or 4 - these do worse, surprisingly).
